gradle/quality.gradle

```diff
@@ -0,0 +1,42 @@
+apply plugin: 'findbugs'
+apply plugin: 'checkstyle'
+apply plugin: 'pmd'
+
+tasks.withType(FindBugs) {
+  excludeFilter = file("$rootProject.projectDir/config/findbugs/excludeFilter.xml")
+}
+
+// ignore test source for checkstyles and pmd
+checkstyle.sourceSets = [sourceSets.main]
+pmd.sourceSets = [sourceSets.main]
+
+pmd {
+    ruleSets = [
+            'java-basic',         // need brandon's http config to add in
```

Comment:
Brandon was the developer at hackweek working on the http config.  It is resolved and comment removed.